deploy/playbooks/roles/common/tasks/ssl.yml

```diff
@@ -0,0 +1,99 @@
+---
+
+- name: ensure ssl certs directory
+  action: file dest=/etc/ssl/certs state=directory
+  sudo: true
+
+- name: ensure ssl private directory
+  action: file dest=/etc/ssl/private state=directory
+  sudo: true
+
+- name: check for SSL cert
+  stat: path=/etc/ssl/certs/{{ fqdn }}-bundled.crt
+  ignore_errors: true
+  register: ssl_cert
+  sudo: true
+
+- name: check for SSL key
+  stat: path=/etc/ssl/private/{{ fqdn }}.key
+  ignore_errors: true
+  register: ssl_key
+  sudo: true
+
+- name: copy SSL cert
+  copy:
+    src: "{{ ssl_cert_path }}"
+    dest: "/etc/ssl/certs/{{ fqdn }}-bundled.crt"
+    mode: 0777
+  when:
+    - ssl_cert.stat.exists == false
```

Comment:
I believe you could set ``force: no`` on the ``copy`` module to avoid overwriting this if it already exists. We could then remove the use of ``stat`` for both the key and cert.